### PR TITLE
Fix XDebug error handler

### PR DESCRIPTION
**Pull request by samdark**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #13671

### CODE REVIEW

File: framework/base/Security.php [hunk 1]
```diff
@@ -273,6 +273,14 @@ protected function decrypt($data, $passwordBased, $secret, $info)
      */
     public function hkdf($algo, $inputKey, $salt = null, $info = null, $length = 0)
     {
+        if (function_exists('hash_hkdf')) {
```
Comment (SilverFire) [inline framework/base/Security.php]: Why are these lines in diff?

**Review by SilverFire — CHANGES_REQUESTED** [review]

**Review by SilverFire — APPROVED** [review]

**Review by dynasource — APPROVED** [review]
shouldnt we move formatting logic to an object for reuse?

**Comment by samdark** [comment]
@dynasource where are we going to reuse it?

**Comment by dynasource** [comment]
no idea ;) but all that logic in the view file does not feel right

**Comment by samdark** [comment]
@dynasource done.

**Pull request merged**